lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -51,11 +53,13 @@ def commands
       "db_export"     => "Export a file containing the contents of the database",
       "db_nmap"       => "Executes nmap and records the output automatically",
       "db_rebuild_cache" => "Rebuilds the database-stored module cache",
+      "analyze"       => "Analyze database information about a specific address or address range",
```

Comment:
Minor: Could this be more descriptive? Analyze to determine what exactly?

Reply:
See PR description, I left this vague as to not restrict ideas on expanding what can be reported as results.